app/javascript/components/pages/EventShow.js

```diff
@@ -6,13 +6,31 @@ class EventShow extends Component {
     this.state = {
       loading: true,
       event: undefined,
+      loadingDrink: true,
+      drink: [],
```

Comment:
You can probably initialize it as an empty object and then store the object that represents a drink since you'll always have only one drink